### ISSUE TITLE
Windows plans: Broken dependencies

### ISSUE THREAD

**Issue by guskovd** [habitat-sh/habitat#2635]
It seems that dependencies are broken when building packages with instructions
$pkg_deps

for example:
```powershell
hab pkg install core/rust
hab pkg exec core/rust rust
```

error: 
```powershell
✗✗✗
✗✗✗ Cannot find package: core/visual-cpp-redist-2013/12.0.30501/20170206154311
✗✗✗
````

**Comment by guskovd** [comment]
It seems that this happens if there are 2 or more elements in $pkg_deps

**Comment by nellshamrell** [comment]
This will be fixed with #2649.

**Comment by mwrock** [comment]
#2649 will use the `lines()` iterator to iterate the lines of the metadata files in a cross platform manner and use OS specific default delimiters for the `PATH` variable.
